Wireframe/index.html

```diff
@@ -3,11 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <title>wireframe</title>
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+<header>
+  <h1>git</h1>
+<p>distributed-even-if-your-workflow-is'nt</p>
+</header>
+<main>
+<div class="git">
+  <article>
+<img src="https://git-scm.com/images/logo@2x.png"class="logo"alt="picture of git">
+<h2>What is Git?</h2>
+<p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.</p>
+   <a href="https://git-scm.com/">  <button> Read More</button></a>
+</article>
+</div>
+<div class= "merge">
+    <div class="developer">
+      <article>
+      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTdCI04wfV2rcwpHP7GI1C95iq_eHzI6sZIyQ&s"class="dev"alt="picture of task tracking system ">
+      <h2>Why do developers need Git?</h2>
+      <p>Git is all about efficiency. For developers, it eliminates everything from the time wasted passing commits over a network connection to the man hours required to integrate changes in a centralized version control system. </p>
+      <a href="https://careerfoundry.com/en/blog/web-development/what-do-developers-use-github-for-heres-why-its-vital/">  <button> Read More</button></a>
+    </article>
+    </div>
+    <div class=" branch"> 
+    <article>
+     
+            <img src="https://wac-cdn.atlassian.com/dam/jcr:a905ddfd-973a-452a-a4ae-f1dd65430027/01%20Git%20branch.svg?cdnVersion=2393" class="pic"alt="picture of git branch">
+            <h2>What is a branch in Git?
+            </h2>
+            <p>Git branches are effectively a pointer to a snapshot of your changes. When you want to add a new feature or fix a bug—no matter how big or how small—you spawn a new branch to encapsulate your changes..</p>
+            <a href="https://www.atlassian.com/git/tutorials/using-branches#:~:text=Git%20branches%20are%20effectively%20a,branch%20to%20encapsulate%20your%20changes.">  <button> Read More</button></a>
+         
+        </article>
+        </div>
+        </div>
+      </main>
+          <footer >
+            <div class="foot">
+            <p>seerat fatima</p>
+            <p><a href="https://github.com/">GitHub</a></p>
```

Comment:
Just a slight nitpick here, but if you use tag, you could technically remove the surrounding

tag and just use to have your link there. More precisely, it's not a necessity to wrap all your anchor () tags in paragraph tags in HTML.